src/Tools/countAccepted.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import pathlib
+
+def count_papers(path):
+    #Path koden som är nedan fungerar, men kanske är bättre att bara ta emot path
+    #path = pathlib.Path.cwd()
+    #path = path.parent.parent / ('data' + path.suffix) / ('meta.csv' + path.suffix)
+    data = pd.read_csv(path)
+    accepted_papers = data.accepted.value_counts()[1]
```

Comment:
Is it guaranteed that 'True' is always the second element?

Wouldn't it be safer with

accepted_papers = data.accepted.value_counts().loc[True]? :) 

Actually I think this code will fail, since value_counts is ordered after the most frequent item. So accepted_papers will always be equal to the most frequent item, even if reject is most frequent.

Reply:
Would say that .loc[True] is less ambiguous though ^ ^